cms/models.py

```diff
@@ -2635,3 +2637,104 @@ def get_context(self, request, *args, **kwargs):
                 "HUBSPOT_ENTERPRISE_PAGE_FORM_ID"
             ),
         }
+
+
+class ExternalCoursePagePlatform(models.Model):
+    """
+    Abstract model for platform specific pages
+    """
+
+    platform = models.ForeignKey(
+        Platform, on_delete=models.SET_NULL, null=True, blank=True
+    )
+
+    class Meta:
+        abstract = True
```

Comment:
I think we don't need this abstract model, We can move the single field to the common mixin that we created below CommonExternalCoursePageMixin.